src/helpers/format-data/index.js

```diff
@@ -28,8 +28,8 @@ const endsOnSameDay = (workshop) =>
 export const formatWorkshopDates = (workshop) => {
   if (workshop && workshop.attributes && workshop.attributes.data_inicio && workshop.attributes.data_fim) {
     return endsOnSameDay(workshop)
-      ? formatDate(new Date(workshop.attributes?.data_inicio))
-      : `${formatDate(new Date(workshop.attributes?.data_inicio))} 
-      > ${formatDate(new Date(workshop.attributes?.data_fim))}`
+      ? formatDate(new Date(`${workshop.attributes?.data_inicio}T00:00:00-0300`))
+      : `${formatDate(new Date(`${workshop.attributes?.data_inicio}T00:00:00-0300`))} 
```

Comment:
Essa correção está se repetindo em tudo que é canto. Não seria melhor deixar a função `formatDate` fazer isso internamente? Continuamos passando a string de data para ela e ela concatena a informação faltante, assim não precisamos repetir a mesma coisa toda vez que a função for chamada.